README.md

```diff
@@ -109,6 +109,11 @@ System.config({
 
 Plugins handle alternative loading scenarios, including loading assets such as CSS or images, and providing custom transpilation scenarios.
 
+#### Usage
+
+1. First, you should install SystemJS plugin by issuing command `jspm install plugin-name`. For example, you are going to use **CSS** plugin to load css files, then you install it with `jspm install css`.
```

Comment:
Actually, this is about JSPM, which is a separate product from JSPM.  This sort of documentation should go over in the JSPM repo.  Item 2 is relevant here though, but if you drop 1 then you should probably remove the numbers.